changelog/2701.added.md

```diff
@@ -0,0 +1,2 @@
+Add termios `OutputFlags::OFILL` for Linux, Andord, Aix, Cygwin, Fuchsia, Haiku, 
```

Comment:
Android 😅

Sorry I read the doc before, but it wasn't clear rust was the format of the Changelog for new items

Reply:
> Android 😅

Thanks for catching it


> but it wasn't clear rust was the format of the Changelog for new items

What do you mean by "rust was the format of the Changelog" 👀, we use Markdown